inhouse-init.sql

```diff
@@ -97,8 +97,7 @@ CREATE TABLE "public"."players" (
     "name" character varying(50) NOT NULL,
     "win" integer NOT NULL,
     "loss" integer NOT NULL,
-    "sp" integer NOT NULL,
-    "show_rank" boolean
+    "sp" integer NOT NULL
 );
```

Comment:
if you changed tables for allowing people outside of `players` make sure we add it to this file too pls.